nabl2.build/pom.xml

```diff
@@ -28,6 +28,7 @@
     <module>../nabl2.shared</module>
     <module>../nabl2.shared.eclipse</module>
     <module>../nabl2.test</module>
+    <module>../shared.primitives</module>
```

Comment:
Since the project is specific to renaming, I suggest to call it `renaming.java` instead.

Reply:
Done